README.md

```diff
@@ -28,6 +28,7 @@ Nodes in the graph are just simple strings with optional data associated with th
  - `directDependenciesOf(name)` - get an array containing the direct dependencies of the specified node
  - `directDependantsOf(name)` (aliased as `directDependentsOf`) - get an array containing the nodes that directly depend on the specified node
  - `overallOrder(leavesOnly)` - construct the overall processing order for the dependency graph. If `leavesOnly` is true, only nodes that do not depend on any other nodes will be returned.
+ - `entryNodes()` - get an array containing the nodes that don't have any depedencies.
```

Comment:
I don't think this is correct- it's an array of nodes that have no dependants (i.e. nothing depends on them)

(also there's a typo - `depedencies`)